docs/docfx.json

```diff
@@ -62,7 +62,7 @@
             ]
         }],
         "xref": [
-            "../tools/msdn.4.5.2/content/msdn.4.5.2.zip"
+            "https://learn.microsoft.com/en-us/dotnet/.xrefmap.json"
```

Comment:
We should do this in the main Akka.NET repo as well - uses the same system (I would love to get off of the old FAKE there too)